sdk/communication/azure-communication-administration/azure/communication/administration/_phone_number_administration_client.py

```diff
@@ -444,10 +444,21 @@ def begin_reserve_phone_numbers(
     ):
         # type: (...) -> LROPoller[PhoneNumberReservation]
         """Begins creating a phone number search to reserve phone numbers.
-        Caller must provide either options, or continuation_token keywords to use the method.
-        If both options and continuation_token are specified, only continuation_token will be used to
-        restart a poller from a saved state, and keyword options will be ignored.
-        :keyword azure.communication.administration.CreateSearchOptions options: reservation options.
+        Caller must provide one of the following:
+         (1) all of keywords display_name, description, phone_plan_ids, area_code, quantity if all the phone plans
+         to reserve are toll-free plans.
+         (2) all of keywords display_name, description, phone_plan_ids, area_code, quantity, location_options
+         if at least one phone plan to reserve is not toll-free plans.
+         (3) only keyword continuation_token to restart a poller from a saved state.
+        If both continuation_token and other keywords are specified, only continuation_token will be used to
+        restart a poller from a saved state, and other keywords will be ignored.
+        :keyword str display_name: display name of the search.
+        :keyword str description: description of the search.
+        :keyword list[str] phone_plan_ids: the plan subtype ids from which to create the search.
+        :keyword str area_code: the area code from which to create the search.
+        :keyword int quantity: the quantity of phone numbers to request.
+        :keyword list[~azure.communication.administration.models.LocationOptionsDetails] location_options:
+            the location options of the search.
```

Comment:
Do we need to add this change into changelog?

Reply:
Sure updated.